src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,54 @@
+import './Footer.css';
+import logoImage from '../../assets/icons/Logo.png';
+import instLogo from '../../assets/icons/inst.png';
+import twitterLogo from '../../assets/icons/twitter.png';
+import youtubeLogo from '../../assets/icons/yt.png';
+
+const Footer = () => {
+    return (
+        <footer className="footer">
+            <div className="footer-content">
+                <div className="footer-left">
+                    <img src={logoImage} alt="Logo" className="footer-logo" />
+                    <p>Takeaway & Delivery template for small - medium businesses.</p>
+                </div>
+                <div className="footer-right">
+                    <div className="footer-column">
+                        <h4>COMPANY</h4>
+                        <ul>
+                            <li>Home</li>
+                            <li>Order</li>
+                            <li>FAQ</li>
+                            <li>Contact</li>
+                        </ul>
```

Comment:
It would be better to make all footer option as array. And all of the should be a clickable link. they could lead to the https://www.google.com/